src/extensions/IExtensionUtilities.ts

```diff
@@ -0,0 +1,9 @@
+import { TemplateCompiler } from '../compilation/templateCompiler';
+import { DocxParser } from '../office';
+import { XmlParser } from '../xml';
+
+export interface IExtensionUtilities {
```

Comment:
Please remove the leading "I".

Naming interface in TypeScript is always an internal question for me. 
While in languages such as C# it's an easy choice since you always should start your interfaces with a captial I, in TypeScript I find it many times more fitting to exclude it - checkout the official documentation for instance: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html#interfaces.

What I usually go with is:
- If the interface describe capabilities, such as IEnumerable, IPrintable and so on I use the leading I.
- If the interface only describe the form of an object (only properties/fields) I usually skip the leading I (for instance, see [PluginUtilities](https://github.com/alonrbar/easy-template-x/blob/master/src/plugins/templatePlugin.ts#L5) and [templateData](https://github.com/alonrbar/easy-template-x/blob/master/src/templateData.ts).

Reply:
Agreed.